src/components/avatars/OcAvatarFederated.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <oc-avatar-item
+    icon="filter_drama"
+    icon-color="#5AAB9F"
+    :name="name"
+    :accessible-label="accessibleLabel"
+  ></oc-avatar-item>
+</template>
+
+<script>
+export default {
+  name: "OcAvatarFederated",
+  status: "ready",
+  release: "2.1.0",
```

Comment:
Please set `status` to `review` and release to an empty value. Will be updated during release.